app/uk/gov/hmrc/vatapi/filters/HeaderValidatorFilter.scala

```diff
@@ -28,6 +28,9 @@ import scala.concurrent.Future
 
 class HeaderValidatorFilter @Inject()(implicit val mat: Materializer, controllerConfiguration: ControllerConfiguration) extends Filter with HeaderValidator {
 
+  protected def executionContext: scala.concurrent.ExecutionContext = ???
+  def parser: play.api.mvc.BodyParser[play.api.mvc.AnyContent] = ???
```

Comment:
These will throw exceptions if ever used.

Reply:
We are not using this at all... and don't want to touch the Filters. so just want to keep them as it is